frontend/src/components/HomePageSections/Hero.tsx

```diff
@@ -0,0 +1,111 @@
+import { useState, useEffect } from 'react'
+import { Link } from 'react-router-dom'
+import HeroImage from '@/assets/HomePage/HeroImage.webp'
+import { Display } from '@/components/ui/Typographies'
+import { CTAButton } from '@/components/ui/cta-button'
+
+const TOTAL_SLIDES = 7;
+const SLIDE_DURATION = 8000; // 8 seconds
+//Changing SLIDE_DURATION will also require changing index.css to match the animation duration
+
+const SLIDE_CONTENT = [
+    "Advocating for and promoting the values and goals of the United Nations within Vancouver.",
+    "Building bridges between communities through global understanding and cooperation.",
+    "Empowering youth to become tomorrow's leaders in sustainable development.",
+    "Promoting human rights and social justice for all members of our community.",
+    "Fostering dialogue and collaboration on critical global challenges.",
+    "Supporting the United Nations Sustainable Development Goals locally.",
+    "Creating opportunities for engagement with international issues and perspectives."
+];
+
+export default function Hero() {
+    const [activeSlide, setActiveSlide] = useState(0);
+    const [key, setKey] = useState(0);
+
+    useEffect(() => {
+        const interval = setInterval(() => {
+            setActiveSlide((current) => (current + 1) % TOTAL_SLIDES);
+            setKey((prev) => prev + 1);
+        }, SLIDE_DURATION);
+
+        return () => clearInterval(interval);
+    }, []);
+    return (
+        <section 
+            id="hero" 
+            className="relative w-full h-[500px] md:h-[610px] flex flex-col items-center pt-20 md:pt-32 pb-10 md:pb-14 px-6 md:px-20 lg:px-40"
+        >
+            {/* Background Image and Overlay */}
+            <div aria-hidden="true" className="absolute inset-0 pointer-events-none overflow-hidden">
```

Comment:
```suggestion
            <div aria-hidden="true" className="absolute inset-0 pointer-events-none overflow-hidden">
``` 

why is the aria hidden here?